.github/workflows/tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    env:     
+      FLOW_TARGET_VERSION: 6.3
+      FLOW_CONTEXT: Testing
+      FLOW_FOLDER: ../flow-base-distribution
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Update Composer
+      run: |
+        sudo composer self-update
+        composer --version
+        
+    # Directory permissions for .composer are wrong, so we remove the complete directory
+    # https://github.com/actions/virtual-environments/issues/824
```

Comment:
https://github.com/actions/virtual-environments/issues/824 seems resolved, so maybe this workaround is no longer necessary?